### DISCUSSION TITLE
Disappointing. Way too expensive. Not free as advertised.

### DISCUSSION THREAD

**Discussion by Panocha** [streetwriters/notesnook#6305] [General]
Not free as stated; "Available for free on all your devices".
Only an extremely, unusable crippled trial version is free.
I was hoping for a Apple Notes replacement, that could be cross-platform, but this isn't it.
$50 a year is a lot. I will keep watching for a real free open source replacement.
Even the most basic function, like being able to copy and paste images into documents, or paste web pages doesn't work, it's paid only, and expensive.
I really hate when they try to draw you in dishonestly. Waste so much time, messing with an app that doesn't work as advertised.
It's kind of shady. If they would make good on what they say, then offer extra paid services, that would be ok, but that's not the case here. Again, the most basic advertised as free features do not work, and it's basically unusable as an Apple Notes type of an application, as shown on their main page for free.

This is simply not true; "Available for free on all your devices
Enjoy **seemless** sync anywhere whether you are on mobile, desktop or tablet.",
Every screenshot they show has images in the notes, which cannot be done.

The above was a copy-paste (https://notesnook.com), and they apparently don't have spell check either.
But again, almost all of the useful features as advertised from their own page as free, do NOT work. It seems like almost everything on their homepage is false click bait, designed to get you in, waste your time, and hope you will pay.

**Comment by thecodrr** [comment]
> Not free as stated; "Available for free on all your devices".

You have excluded the relevant context around this line. The above mentioned line is talking about the _sync_ which _is_ free and available on all your devices.

>  Only an extremely, unusable crippled trial version is free.

I think your expectation is that Notesnook should be 100% free (which is unrealistic) but that just means Notesnook is not for you.

>  I was hoping for a Apple Notes replacement, that could be cross-platform, but this isn't it.
> $50 a year is a lot.

Okay. No one is forcing you to pay. The free version comes with no strings attached (you don't even have to create an account).

> I will keep watching for a real free open source replacement.

I wish you best of luck. It is clear to me that Notesnook is not the right fit for your use case.

> Every screenshot they show has images in the notes, which cannot be done.

I don't think mentioning under each and every image that X, Y, or Z feature is not available in the free plan is realistic. It is expected that the user will do their due diligence and check the pricing page where everything is stated in much more detail.

> they apparently don't have spell check either.

Snide remarks such as these help no one. I understand your point of view but please do keep a check on your tone. Being respectful is only decent.

> It seems like almost everything on their homepage is false click bait, designed to get you in, waste your time, and hope you will pay.

You are starting to sound like a troll now.
